### ISSUE TITLE
Play Pause button needs improvement to communicate states

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#299]
There are a few different types of binary logic interactions in HTML. Traditionally, we have toggle buttons (pressed and not pressed), checkboxes (checked and not checked), and 2 radiobuttons (only one of which can be selected). 

The labels (or names of the elements) and their states combine to communicate the success of the interaction. Thus, it is crucial to think carefully what the interaction is and how we want it to be communicated because depending on the code the interaction is communicated in different ways and can be confusing if not done right.

This article, [Building Inclusive Toggle Buttons](https://www.smashingmagazine.com/2017/09/building-inclusive-toggle-buttons/) by Heydon Pickering, explains the these differences very well, and specifically uses a Play/Pause button as an example.

A Play/Pause toggle button communicates it's change through the name change, so we have to make sure this name change is communicated to AT.

In the Ad/Remove Wall button in BASE, we did this with aria-live alerts.

In this issue, we can work out a design pattern for PhET toggle buttons that communicate their  changed state through a name change.

**Comment by terracoda** [comment]
This issue related to https://github.com/phetsims/scenery-phet/issues/575 and possible related to https://github.com/phetsims/molecules-and-light/issues/295.

**Comment by terracoda** [comment]
The visual I imagined for option 3 (down the road if we ever go there), might be something like these states:
![Screen Shot 2020-03-12 at 10 07 43 PM](https://user-images.githubusercontent.com/1902116/76579255-6fbd8580-64ae-11ea-8818-2fcc34d12fb7.png)
![Screen Shot 2020-03-12 at 10 07 51 PM](https://user-images.githubusercontent.com/1902116/76579250-6d5b2b80-64ae-11ea-9b80-326d2d0b19c3.png)
These images were taken from https://scottaohara.github.io/a11y_styled_form_controls/src/toggle-button-switch/, but the actual buttons in the demo are not screen reader accessible, only keyboard accessible.

**Comment by terracoda** [comment]
@jessegreenberg, I have created new UI context responses for the following:
- Light Source Toggle Button
- Pause/Play Button
- Step Forward Button

They may be too sim specific as you mentioned in #311 
I think the responses help clarify the interactions, but there may be other solutions that are more general.

Let's discuss at Tuesday's a11y design meeting.

**Comment by jessegreenberg** [comment]
I read through the issue and all sounds good @terracoda. By the end of meeting today we didn't decide how to proceed with this issue. Removing my assignment for now, but feel free to reasign when its ready for more work.

**Comment by terracoda** [comment]
I'll re-assign after I discuss with @emily-phet. Thanks!

**Comment by terracoda** [comment]
@jessegreenberg, I spoke with @emily-phet about how to move forward on the Time Control Node and UI Context responses for this widget. Here's the two-pronged approach that  mentioned on slack:
1. First implement the UI context responses for Pause/Play and Ste Forward buttons so we have an accessible non-visual experience, and then
2. work on the new option that would allow disabling the Pause/Play button for Timing Control Node. Do this after the UI responses are implemented.

If  changes to the Time Control Node are done before RC we can consider using the new option and removing (or generalizing/adjusting) the UI context responses AND/OR the help text for this common group of controls.

Note: I already added some potential generalized UI context responses. Not sure if they are super great, but they are drafted.

@jessegreenberg, I'll make 2 new issues for these options.

**Comment by terracoda** [comment]
Just noting this other issue where a lot of discussion happened around toggle buttons
https://github.com/phetsims/a11y-research/issues/137#issuecomment-482796376

**Comment by terracoda** [comment]
We decided last week to use context responses to scaffold non-visual users in using the Pause/Play button. Implementation of those will be tracked in #317.

**Comment by terracoda** [comment]
Closing this issue.

**Comment by terracoda** [comment]
We may need to

**Comment by terracoda** [comment]
We can re-open this issue if we need to specifically address how we communicate the states of the Pause/Play button that is currently communicating its state through a label change.
